avro/swagger/src/main/java/com/cerner/beadledom/avro/SwaggerAvroModelConverter.java

```diff
@@ -119,49 +116,39 @@ protected String getName(Schema schema) {
    * determined merely by looking at the schema. It may also be used recursively to build collection
    * and union types.
    *
-   * @return the parsed property, or null if it cannot/should not be represented in the Swagger
-   *     model
+   * @return the parsed property, or null if it cannot/should not be represented in the Swagger model
    */
   @Nullable
-  protected ModelProperty parseSchema(Schema schema) {
+  protected Property parseSchema(Schema schema) {
     switch (schema.getType()) {
       case RECORD:
-        return simpleProperty(getName(schema), schema.getFullName());
+        // todo PropertyBuilder.build() returning null
+        return null;
+//        Property record = PropertyBuilder.build(getName(schema), schema.getFullName(), Collections.emptyMap());
+//        record.setRequired(true);
+//        return record;
       case ENUM:
         // TODO may wish to include the enum's documentation in the field documentation, since it
```

Comment:
Does this still apply?

Reply:
Yea, just found that record isn't working

Reply:
partially fixed 32034b099b

Reply:
b2ed9351

Reply:
sweet, can we delete this todo then?

Reply:
They are already deleted in the latest version.